en/firefox/privacy/passwords.ftl

```diff
@@ -23,7 +23,7 @@ privacy-passwords-make-your-answers = Make your answers to security questions ju
 privacy-passwords-use-a-password = Use a password manager to make creating and remembering passwords easier
 privacy-passwords-use-twofactor-authentication = Use “two-factor authentication” wherever you can
 privacy-passwords-its-hard-out = It’s hard out there for a password
-privacy-passwords-most-logins-today = Most logins today are protected by a password. If an attacker can get your password, he can access your account and do anything you could do with that account. So when you ask how secure your account is, you should really be thinking about how safe your password is. And that means you have to think about all the different ways that an attacker could access your account’s password:
+privacy-passwords-most-logins-today = Most logins today are protected by a password. If an attacker can get your password, they can access your account and do anything you could do with that account. So when you ask how secure your account is, you should really be thinking about how safe your password is. And that means you have to think about all the different ways that an attacker could access your account’s password:
```

Comment:
@peiying2 I saw here you asked for a different string ID: https://github.com/mozilla/bedrock/commit/266bca0b4c36f04f98c1aec0bef57c96ad2470c5#commitcomment-47561879

Please see my comment /question, which I'll repost here below:

> Hmm, I'm not sure I quite follow why we need to change the string ID for a small edit like this? The overall meaning of the string hasn't changed, and locales are free to change the language. Or have too many locales also used a non-neutral term?